<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WaveWgtClass</class>
 <widget class="QWidget" name="WaveWgtClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1225</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WaveWgt</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_leftCh">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>leftCh</string>
     </property>
     <layout class="QVBoxLayout" name="VLayout_LeftCh"/>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_rightCh">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>rightCh</string>
     </property>
     <layout class="QVBoxLayout" name="VLayout_rightCh"/>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>pannel</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_cht_load_startTime">
            <property name="text">
             <string>载入开始时间</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTimeEdit" name="timeEdit_chtLoadStartTime">
            <property name="displayFormat">
             <string>mm:ss.zzz</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_cht_load_endTime">
            <property name="text">
             <string>载入结束时间</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTimeEdit" name="timeEdit_chtLoadEndTime">
            <property name="displayFormat">
             <string>mm:ss.zzz</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_reloadCht">
            <property name="text">
             <string>重新加载时间范围</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_loadToDataCenter">
            <property name="text">
             <string>存储到DCenter</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_loadAll">
            <property name="text">
             <string>加载全部</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_reloadState">
            <property name="text">
             <string>重新载入状态：</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_windowStartTime">
            <property name="minimumSize">
             <size>
              <width>140</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>140</width>
              <height>20</height>
             </size>
            </property>
            <property name="text">
             <string>窗口开始时间:00:00:00</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QScrollBar" name="HScrollBar_windowPos">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>20</height>
             </size>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_windowEndTime">
            <property name="minimumSize">
             <size>
              <width>140</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>140</width>
              <height>20</height>
             </size>
            </property>
            <property name="text">
             <string>窗口结束时间:00:00:00</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_windowLength">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>窗口长度：60s</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="hSlider_windowScale">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>10</number>
            </property>
            <property name="pageStep">
             <number>1</number>
            </property>
            <property name="sliderPosition">
             <number>5</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="tickPosition">
             <enum>QSlider::TicksAbove</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_currentTime">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>当前时间：00:00:00</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="hSlider_currentTime">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btn_stop">
            <property name="text">
             <string>stop</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_play">
            <property name="text">
             <string>play</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_pause">
            <property name="text">
             <string>pause</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>16777215</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="VLayout_volumnBar">
         <property name="leftMargin">
          <number>2</number>
         </property>
         <property name="topMargin">
          <number>2</number>
         </property>
         <property name="rightMargin">
          <number>2</number>
         </property>
         <property name="bottomMargin">
          <number>2</number>
         </property>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
